solr/test-framework/src/java/org/apache/solr/cloud/AbstractFullDistribZkTestBase.java

```diff
@@ -1981,29 +1981,18 @@ protected void destroyServers() throws Exception {
     ExecutorService customThreadPool =
         ExecutorUtil.newMDCAwareCachedThreadPool(new SolrNamedThreadFactory("closeThreadPool"));
 
-    customThreadPool.submit(() -> IOUtils.closeQuietly(commonCloudSolrClient));
+    customThreadPool.execute(() -> IOUtils.closeQuietly(commonCloudSolrClient));
 
-    customThreadPool.submit(() -> IOUtils.closeQuietly(controlClient));
+    customThreadPool.execute(() -> IOUtils.closeQuietly(controlClient));
 
-    customThreadPool.submit(
-        () ->
-            coreClients.parallelStream()
-                .forEach(
-                    c -> {
-                      IOUtils.closeQuietly(c);
-                    }));
+    customThreadPool.execute(() -> coreClients.parallelStream().forEach(IOUtils::closeQuietly));
```

Comment:
shouldn't we avoid parallelStream?  Could loop these to submit them to customThreadPool.  Same for below.

Reply:
Good point - even though this uses IOUtils#closeQuietly to avoid propagating the exception, it does make sense to be consistent about the usage of parallelStream.